### DISCUSSION TITLE
Set Model Match number in Device Options

### DISCUSSION THREAD

**Discussion by Darren-A11att** [ExpressLRS/ExpressLRS-Configurator#615] [Ideas]
As a Configurator User I want to set the Model ID when flashing ELRS to a Receiver so that I don't have to do it via the LUA or over Wifi

- When a receiver device is selected, Model ID is shown in Device Options
- Ticking Model ID reveals a value field where I can enter the Model ID
- When I flash the receive it sets the model ID
- If I don't tick model id, the existing model id remains

**Comment by jurgelenas** [comment]
Thank you @Darren-A11att for this idea. I have closed it temporary and converted to the discussion topic.

**Comment by Krizon4** [comment]
This feature is on top of my wishlist too. Lack of any options to configure ModelID during flash makes the whole feature real pain to use.
Along with the unability to override and set model ID from LUA.

Is there any way to motivate someone to actually implement this ?
